ast-checks/src/no-uri-literals.rs

```diff
@@ -0,0 +1,71 @@
+use codespan::{FileId, Files};
+use common_structs::{NixpkgsHammerMessage, SourceLocation};
+use rnix::{types::*, SyntaxKind::*};
+use std::{collections::HashMap, env, error::Error, fs};
+use tree_utils::walk_kind;
+
+mod common_structs;
+mod tree_utils;
```

Comment:
I am getting warnings about dead code: warning: function is never used: `walk_keyvalues_filter_key`

Reply:
Managed to resolve this by making the modules into a crate. This also allows us to get rid of the `bin` listings in `Crate.toml`.